utils.py

```diff
@@ -1012,349 +769,6 @@ def quote_field_name(f):
-        """
-        condition = functions.to_expr(condition)
-        then = functions.to_expr(then)
-        self._unify_type(then.dtype)
-        self._cases.append((condition, then))
-        return self
-
-    def default(self, then):
-        """Finish the case statement by adding a default case.
-
-        Notes
-        -----
-        If no condition from a :meth:`~.CaseBuilder.when` call is ``True``,
-        then `then` is returned.
-
-        Parameters
-        ----------
-        then : :class:`.Expression`
-
-        Returns
-        -------
-        :class:`.Expression`
-        """
-        then = functions.to_expr(then)
-        if len(self._cases) == 0:
-            return then
-        self._unify_type(then.dtype)
-        return self._finish(then)
-
-    def or_missing(self):
-        """Finish the case statement by returning missing.
-
-        Notes
-        -----
-        If no condition from a :meth:`.CaseBuilder.when` call is ``True``, then
-        the result is missing.
-
-        Parameters
-        ----------
-        then : :class:`.Expression`
-
-        Returns
-        -------
-        :class:`.Expression`
-        """
-        if len(self._cases) == 0:
-            raise ExpressionException("'or_missing' cannot be called without at least one 'when' call")
-        from hail.expr.functions import null
-        return self._finish(null(self._ret_type))
-
-
-def case():
-    """Chain multiple if-else statements with a :class:`.CaseBuilder`.
-
-    Examples
-    --------
-    .. doctest::
-
-        >>> x = functions.capture('foo bar baz')
-        >>> expr = (functions.case()
-        ...                  .when(x[:3] == 'FOO', 1)
-        ...                  .when(x.length() == 11, 2)
-        ...                  .when(x == 'secret phrase', 3)
-        ...                  .default(0))
-        >>> eval_expr(expr)
-        2
-
-    See Also
-    --------
-    :class:`.CaseBuilder`
-
-    Returns
-    -------
-    :class:`.CaseBuilder`.
-    """
-    return CaseBuilder()
-
-
-def switch(expr):
-    """Build a conditional tree on the value of an expression.
-
-    Examples
-    --------
-    .. doctest::
-
-        >>> csq = functions.capture('loss of function')
-        >>> expr = (functions.switch(csq)
-        ...                  .when('synonymous', 1)
-        ...                  .when('SYN', 1)
-        ...                  .when('missense', 2)
-        ...                  .when('MIS', 2)
-        ...                  .when('loss of function', 3)
-        ...                  .when('LOF', 3)
-        ...                  .or_missing())
-        >>> eval_expr(expr)
-        3
-
-    See Also
-    --------
-    :class:`.SwitchBuilder`
-
-    Parameters
-    ----------
-    expr : :class:`.Expression`
-        Value to match against.
-
-    Returns
-    -------
-    :class:`.SwitchBuilder`
-    """
-    return SwitchBuilder(functions.to_expr(expr))
-
 
 def merge_TStructs(s):
     """
```

Comment:
`replace_vds_variant_schema()` hasn't been ported over to Hail2 syntax (uses `annotate_variants_expr()`)